src/sentry/static/sentry/app/stores/pluginsStore.jsx

```diff
@@ -0,0 +1,97 @@
+import Reflux from 'reflux';
+
+import PluginActions from '../actions/pluginActions';
+
+const PluginsStore = Reflux.createStore({
+  getInitialState() {
+    return this.getState();
+  },
+
+  getState() {
+    let {
+      //eslint-disable-next-line no-unused-vars
+      plugins,
+      ...state
+    } = this.state;
+
+    return {
+      ...state,
+      plugins: this.plugins ? Array.from(this.plugins.values()) : this.plugins,
```

Comment:
I don't think we should allow plugins to be `null`, how about:
`plugins: this.plugins ? Array.from(this.plugins.values()) : []`

Reply:
`null` value is being used to mean that the store is in a loading state, vs having an explicit state for it. Would adding comments make this less confusing?

Reply:
I see. Is it necessary for the components that use this store to know whether it's in a loading state?